aptos-move/aptos-release-builder/data/release.yaml

```diff
@@ -23,6 +23,7 @@ update_sequence:
         - bls12381_structures
         - ed25519_pubkey_validate_return_false_wrong_length
         - struct_constructors
+        - signature_checker_v2
```

Comment:
This seem to come from an old version. Can you rebase?

Reply:
Updated